code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -125,7 +125,7 @@
 			dir_shots.Trigger(target = target)
 
 /mob/living/simple_animal/hostile/megafauna/colossus/proc/telegraph()
-	for(var/mob/viewer as anything in viewers(10, src))
+	for(var/mob/viewer as anything in viewers(DEFAULT_SIGHT_DISTANCE, src))
```

Comment:
It's not much but on lavaland it's probably important for people this 3 tile distance away to know that colossus is about to attack

Reply:
i will test this to see if its actually getting viewers at 10. curious